src/components/PostPageVolunteerHeader/index.css

```diff
@@ -0,0 +1,18 @@
+.headerPicture {
+  width: 100%;
+  height: auto;
```

Comment:
i think this is unnecessary, even if it was removed it would give the same result because it is auto